src/com/facebook/buck/ide/intellij/DefaultIjLibraryFactory.java

```diff
@@ -154,6 +161,12 @@ public void apply(
           .ifPresent(
               sourcePath -> library.addSourceJars(libraryFactoryResolver.getPath(sourcePath)));
       arg.getJavadocUrl().ifPresent(library::addJavadocUrls);
+
+      Path aarUnpackPath = BuildTargets.getScratchPath(
+          projectFilesystem, targetNode.getBuildTarget(), "__unpack_%s#aar_unzip__");
+      if (aarUnpackPath != null) {
```

Comment:
it should probably always be non-null since we're unpacking the aar in this rule, but since it was looking at file paths, I was defensive since the Layout Preview is more optional than generating the project